mozilla/blurts-server
# add global client error handling per Sentry docs

# References: 
Jira: MNTOR-2671



# Description

Add global client error handling per Sentry docs. This should catch any uncaught exceptions during rendering.

# How to test

Injecting an exception into a client-side component is the simplest way I can think to test:

```diff
diff --git a/src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/View.tsx b/src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/View.tsx
index 1a3f7c1ff..acab46891 100644
--- a/src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/View.tsx
+++ b/src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/View.tsx
@@ -77,6 +77,7 @@ export type TabData = {
 };
 
 export const View = (props: Props) => {
+  throw new Error("test123");
   const l10n = useL10n();
   const recordTelemetry = useTelemetry();
   const countryCode = useContext(CountryCodeContext);
```

You should see the exception being sent to Sentry in the devtools network tab, and it should show up in the Sentry UI (if you filter on `local`). I just tried and this wfm, stack traces seem to work as well (I believe it's using the public sourcemap files we publish).
